packages/shared/src/utils/platform/platform.ts

```diff
@@ -167,15 +167,27 @@ export const isNavigationFromExternalPlatform = (routing_history: TRoutingHistor
     return false;
 };
 
-export const isDtraderV2Enabled = (is_mobile: boolean) => {
-    const dtrader_v2_enabled_growthbook = Analytics?.getFeatureValue('dtrader_v2_enabled', false);
-    const is_dtrader_v2 =
-        JSON.parse(localStorage.getItem('FeatureFlagsStore') ?? '{}')?.data?.dtrader_v2 ||
-        dtrader_v2_enabled_growthbook;
+export const isDtraderV2MobileEnabled = (is_mobile: boolean) => {
+    const dtrader_v2_enabled_gb = Analytics?.getFeatureValue('dtrader_v2_enabled', false);
+    const is_dtrader_v2_mobile =
+        JSON.parse(localStorage.getItem('FeatureFlagsStore') ?? '{}')?.data?.dtrader_v2_mobile || dtrader_v2_enabled_gb;
 
     return (
-        is_dtrader_v2 &&
+        is_dtrader_v2_mobile &&
         is_mobile &&
         (window.location.pathname.startsWith(routes.trade) || window.location.pathname.startsWith('/contract/'))
     );
 };
+
+export const isDtraderV2DesktopEnabled = (is_desktop: boolean) => {
```

Comment:
isDtraderV2DesktopEnabled  is almost the same as isDtraderV2MobileEnabled. Maybe we can refactor it?